.changeset/slimy-lobsters-jump.md

```diff
@@ -0,0 +1,10 @@
+---
+'@guardian/libs': major
+---
+
+CMP: Implement the Multi-State-Privacy-Agreement in the US
```

Comment:
would you be able to mention which actual export from libs are affected by this change, and what, if anything, consumers need to do?